portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,49 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/** Container class for Form data */
+class FormData {
+  private String nickname; //The alias of the commenter
+  private String comment; // The assocaited comment
```

Comment:
typo: `s/assocaited/associated/` (replace assocaited with associated)

Reply:
Updated in commit 4211bf8